### PR TITLE
Fix typo, become chadributor

### PR DESCRIPTION
**Pull request by buffet**

### CODE REVIEW

**Comment by zbirenbaum** [comment]
@siduck can we screenshot this and put it in the docs under contributions I'm laughing so hard rn.

There's also a random image added that you wouldn't notice unless you look at the code and view the file preview.

2022-02-22-001151_908x618_scrot
.

File: .github/README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+![image](https://user-images.githubusercontent.com/33751841/154978244-5d2841af-4b80-4c1b-8af2-f7f045df63c5.png)
```
Comment (zbirenbaum) [inline .github/README.md:1]: What is this here for? Where did it come from?

**Comment by buffet** [comment]
Whoops, I have no idea how that ended up in there. Gimme a second.

On Feb 22, 2022, 05:15, at 05:15, Zach Birenbaum ***@***.***> wrote:
***@***.*** can we screenshot this and put it in the docs under

**Comment by buffet** [comment]
And that's why you don't use the GitHub's editor, kids.

**Review by siduck — APPROVED** [review]

**Pull request merged**